scripts/create-release-pr.sh

```diff
@@ -69,7 +69,21 @@ fi
 
 if [[ -n $ARTIFACTS_DIR_PATH ]]; then
   # Write PR number to file so that it can be uploaded as an artifact
-  PR_NUMBER=$(gh pr view --json number | jq '.number')
+
+  # Retry getting the number on a timeout in case GitHub is slow to create the PR
+  PR_NUMBER=''
+  for i in {1..7} # 6 + 1 times total
+  do
+    PR_NUMBER=$(gh pr view --json number | jq '.number')
+    if [[ -z $PR_NUMBER ]]; then
```

Comment:
Are you testing that the command returns a result here? If so, should this be `-n`?

Reply:
Whoops, yes!